# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1281396618 14400
#      Mon Aug 09 19:30:18 2010 -0400
# Node ID 53ec464300ef400c43cf6ecbc63a290c8e2f0fbc
# Parent  1bc534956e2992c0c26cd49573ba135edf9e5d9a
__print_parse_opts__.m: Default pstex suffix should be '.ps'.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-08-09  Ben Abbott <bpabbott@mac.com>
+
+	* plot/__print_parse_opts__.m: Default pstex suffix should be '.ps'.
+
 2010-08-08  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Create .DOCSTRINGS quietly and avoid screen clutter
 	during build.
 
 2010-08-08  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Run do_subst macro quietly without cluttering screen 
diff --git a/scripts/plot/__print_parse_opts__.m b/scripts/plot/__print_parse_opts__.m
--- a/scripts/plot/__print_parse_opts__.m
+++ b/scripts/plot/__print_parse_opts__.m
@@ -214,17 +214,17 @@ function arg_st = __print_parse_opts__ (
               "ppm", "ppmraw", "pdflatex"};
 
   suffixes = {"ai", "cdr", "fig", "png", "jpg", ...
               "gif", "pbm", "pbm", "dxf", "mf", ...
               "svg", "hpgl", "ps", "ps", "ps", ...
               "ps", "eps", "eps", "eps", "eps", ...
               "emf", "pdf", "tex", "tex", "tex", ...
               "tex", "tex", ...
-              "tex", "tiff", "tiff", "tikz", "pcx", ...
+              "ps", "tiff", "tiff", "tikz", "pcx", ...
               "pcx", "pcx", "pcx", "pgm", "pgm", ...
               "ppm", "ppm", "tex"};
 
   match = strcmpi (dev_list, arg_st.devopt);
   if (any (match))
     default_suffix = suffixes {match};
   else
     default_suffix = arg_st.devopt;
